objectivec/ort_enums.mm

```diff
@@ -53,6 +53,7 @@
     {ORTTensorElementDataTypeUInt32, ONNX_TENSOR_ELEMENT_DATA_TYPE_UINT32, sizeof(uint32_t)},
     {ORTTensorElementDataTypeInt64, ONNX_TENSOR_ELEMENT_DATA_TYPE_INT64, sizeof(int64_t)},
     {ORTTensorElementDataTypeUInt64, ONNX_TENSOR_ELEMENT_DATA_TYPE_UINT64, sizeof(uint64_t)},
+    {ORTTensorElementDataTypeString, ONNX_TENSOR_ELEMENT_DATA_TYPE_STRING, 32},
```

Comment:
I know I suggested using an arbitrary value for the size here, but on second thought it doesn't really make sense for the `UNDEFINED` and `STRING` types to have a size defined. how about changing `TensorElementTypeInfo::element_size` to `std::optional<size_t>`?